### PR TITLE
Better HP and migration

### PR DESCRIPTION
**Pull request by COil**
* Transferred SQL to the migration
* Added links on HP to avoid typing the API Platform URL
* Other minor cleanup

### CODE REVIEW

File: public/index.php [hunk 1]
```diff
@@ -1,9 +1,11 @@
 <?php
 
+declare(strict_types=1);
+
 use App\Kernel;
 
 require_once dirname(__DIR__).'/vendor/autoload_runtime.php';
 
-return function (array $context) {
+return static function (array $context) {
```
Comment (COil) [inline public/index.php:9]: This is a recommendation of the EA plugin : `// [EA] This closure can be declared as static (better scoping; in some cases can improve performance).` I suppose this is a micro optimization.

Comment (dunglas) [reply public/index.php:9]: Can we stay in sync with upstream Symfony? Maybe can you propose this change to Symfony directly?

File: src/Controller/HomepageController.php [hunk 2]
```diff
@@ -12,7 +14,7 @@ class HomepageController extends AbstractController
     public function index(): Response
     {
        return $this->render('homepage/index.html.twig', [
-            'controller_name' => 'HomepageController',
+            'controller_name' => $this::class,
```
Comment (dunglas) [inline src/Controller/HomepageController.php:17]: Same here, I would keep it as-is for now, but maybe can you contribute this change to MakerBundle?

File: src/Entity/Monster.php [hunk 3]
```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```
Comment (dunglas) [inline src/Entity/Monster.php:3]: Can you remove strict types too please? (Same reason).

**Review by dunglas — CHANGES_REQUESTED** [review]

**Comment by COil** [comment]
Done.

**Comment by COil** [comment]
OK for you @dunglas  ? 🙂

**Review by dunglas — APPROVED** [review]

**Comment by dunglas** [comment]
Thanks!

**Pull request merged**